LoginPlugin/iOS/ZappLoginPluginExample-iOS/ViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ViewController.swift
+//  ZappPluginPlayerExample-iOS
+//
+//  Created by Udi Lumnitz on 19/06/2017.
+//  Copyright © 2017 Applicaster. All rights reserved.
+//
+
+import UIKit
+import ZappPlugins
+import ApplicasterSDK
+
+class ViewController: UIViewController {
```

Comment:
i think this file can also be removed...

Reply:
yep